backend/rollback.go

```diff
@@ -42,11 +42,31 @@ func (b *Backend) RollbackApplication(applicationId string) error {
 		return err
 	}
 
-	tasks, err := b.store.ListTasks(applicationId)
+	tasks, err := b.store.ListTasks(appId)
 	if err != nil {
 		return err
 	}
 
+	go func() error {
```

Comment:
@upccup yes, i haven't known how to use it , i'm designing. it will be used in future.